### PR TITLE
Add contact info for each test result gcs prefix

### PR DESCRIPTION
**Pull request by fejta**
@justinsb @euank @travisnewhouse

### CODE REVIEW

File: buckets.yaml [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# A mapping of bucket URLs to bucket metadata
+# A mapping of bucket URLs to bucket metadata.
+# Please include a contact for new prefixes!
```
Comment (travisnewhouse) [inline buckets.yaml]: Should we prescribe what kind of contact information, e.g., Slack user ID, email?

Comment (travisnewhouse) [inline buckets.yaml]: ... and then it hits me later: GitHub username.

**Comment by euank** [comment]
LGTM

File: buckets.yaml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 #
 # Possible bucket metadata:
 #   prefix: the identifier used for this bucket on the test-history dashboard.
 #   sequential: an optional boolean that indicates whether test runs in this
 #     bucket are numbered sequentially. This is used for an optimization in the
 #     test-history collection phase, and defaults to true.
 
-gs://kubernetes-jenkins/logs/:
+gs://kubernetes-jenkins/logs/:  # @fejta
```
Comment (ixdy) [inline buckets.yaml]: is there a reason we don't just make the owner actual metadata? that way we can potentially use it in tooling?

Comment (spxtr) [inline buckets.yaml]: Agreed. I would rather just add a contact tag.

Comment (fejta) [inline buckets.yaml]: Great idea, done

File: buckets.yaml [hunk 3]
```diff
@@ -1,4 +1,5 @@
-# A mapping of bucket URLs to bucket metadata
+# A mapping of bucket URLs to bucket metadata.
+# Please include a github contact for new prefixes!
 #
 # Possible bucket metadata:
```
Comment (ixdy) [inline buckets.yaml]: add `contact` into this list?

File: buckets.yaml [hunk 4]
```diff
@@ -7,11 +8,15 @@
 #     test-history collection phase, and defaults to true.
 
 gs://kubernetes-jenkins/logs/:
+  contact: "fejta"
   prefix: ""
 gs://kube_azure_log/:
+  contact: "travisnewhouse"
   prefix: "azure:"
-gs://rktnetes-jenkins/logs/:
+gs://rktnetes-jenkins/logs/:  # @euank
```
Comment (euank) [inline buckets.yaml]: strip the comment too for consistency.
The key could also be `contact_github_username` or such, though just specifying it in the comment at the top seems cleaner.

**Comment by spxtr** [comment]
Even more bikeshedding: it might be worth doing "contacts" and having a list. Probably not necessary for now, though.

File: buckets.yaml (hunk 4 above)
Comment (fejta) [inline buckets.yaml]: Done

File: buckets.yaml (hunk 3 above)
Comment (fejta) [inline buckets.yaml]: Done

**Pull request merged**